framework_tvb/tvb/core/services/project_service.py

```diff
@@ -607,7 +604,6 @@ def remove_operation(self, operation_id):
             for dt in reversed(datatypes_for_op):
                 self.remove_datatype(operation.project.id, dt.gid, False)
             dao.remove_entity(Operation, operation.id)
-            self.structure_helper.remove_operation_data(operation.project.name, operation_id)
```

Comment:
Why remove this line?
Removing an operation means cleaning the entire Operation folder, and that is still valid in the new code, right?

Reply:
Readded it.